src/_fastmath.c

```diff
@@ -1342,6 +1352,10 @@ rabinMillerTest (mpz_t n, int rounds, PyObject *randfunc)
 	}
 
 cleanup:
+	for (i = 0; i <= last_tested_cleanup; ++i)
```

Comment:
This will be executed for `last_tested_cleanup = 0`, even if we never reached to a `last_tested_cleanup = i;` line (and, consequentially, we never set `tested[0]`

Reply:
Great catch, Michael! I have fixed this. How does it look now?

Reply:
@nickwilliams-eventbrite this is an issue. You can fix it by making `last_tested_cleanup` signed and set its initial value to `-1`;